nest-city-account/Dockerfile

```diff
@@ -1,4 +1,7 @@
+ARG SHARED_PACKAGE_TAG
+FROM harbor.bratislava.sk/standalone/openapi-clients:${SHARED_PACKAGE_TAG} as openapi-clients
```

Comment:
(optional) If you don't needed as a separate stage or you need it just once, you can use the URL directly in the `--from` option of a `COPY` instruction https://stackoverflow.com/a/76063218

Reply:
It's needed both in build and prod, so I keep it this way. (All this is taken from already existing solution in nest-forms-backend)